src/resourcemgr.cpp

```diff
@@ -99,62 +99,6 @@ bool ResourceMgr::copyResourceAs(const QCString &name,const QCString &targetDir,
           }
         }
         break;
-      case Resource::Luminance:
-        {
-          QCString n = name;
-          n = n.left(n.length()-4)+".png"; // replace .lum by .png
-          const uint8_t *data = res->data;
-          uint16_t width   = (data[0]<<8)+data[1];
-          uint16_t height  = (data[2]<<8)+data[3];
-          ColoredImgDataItem images[2];
-          images[0].name    = n.data();
-          images[0].width   = width;
-          images[0].height  = height;
-          images[0].content = &data[4];
-          images[0].alpha   = nullptr;
-          images[1].name    = nullptr; // terminator
-          writeColoredImgData(targetDir,images);
-          return TRUE;
-        }
-        break;
-      case Resource::LumAlpha:
-        {
-          QCString n = name;
-          n = n.left(n.length()-5)+".png"; // replace .luma by .png
-          const uint8_t *data = res->data;
-          uint16_t width   = (data[0]<<8)+data[1];
-          uint16_t height  = (data[2]<<8)+data[3];
-          ColoredImgDataItem images[2];
-          images[0].name    = n.data();
-          images[0].width   = width;
-          images[0].height  = height;
-          images[0].content = &data[4];
-          images[0].alpha   = &data[4+width*height];
-          images[1].name    = nullptr; // terminator
-          writeColoredImgData(targetDir,images);
-          return TRUE;
-        }
-        break;
-      case Resource::CSS:
```

Comment:
This case was still unused (got a compiler warning for it), so I reintroduced it (without the `navtree.css` specifics).

Reply:
I see in the GitHub Actions indeed a warning:
```
home/runner/work/doxygen/doxygen/src/resourcemgr.cpp:85:12: warning: enumeration value ‘CSS’ not handled in switch [-Wswitch]
   85 |     switch (res->type)
      |       
```
Though, as far as I can tell, the relevant code is never called. The `CSS` is used in the setting `initResources` but in the routine `getAsString` it is not used at all. 
We probably need another follow up PR to handle this.

Reply:
> We probably need another follow up PR to handle this.

Will be hard as the `type` is used at the above mentioned line 85 in the switch statement.